resources/views/layouts/app.blade.php

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
+<html lang="{{ app()->getLocale() }}">
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+
+    <!-- CSRF Token -->
+    <meta name="csrf-token" content="{{ csrf_token() }}">
+
+    <title>{{ config('app.name', 'Laravel') }}</title>
+
+    <!-- Styles -->
+    <link href="{{ asset('css/app.css') }}" rel="stylesheet">
+</head>
+<body>
+    <div id="app">
+        <nav class="navbar navbar-default navbar-static-top">
+            <div class="container">
+                <div class="navbar-header">
+
+                    <!-- Collapsed Hamburger -->
+                    <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#app-navbar-collapse">
+                        <span class="sr-only">Toggle Navigation</span>
+                        <span class="icon-bar"></span>
+                        <span class="icon-bar"></span>
+                        <span class="icon-bar"></span>
+                    </button>
+
+                    <!-- Branding Image -->
+                    <a href="{{ url('/') }}">
+                        <img style="width: 40%;" alt="logo" src="https://www.google.com/a/devmob.com/images/logo.gif?alpha=1&service=google_default">
```

Comment:
When you add a logo to your repository, it's best to add that folder to your assets directory. If for some reason this image might get removed/replaced, your website design might break. Add the image somewhere in your `public` directory and then use it like this:

``` blade
<img style="width: 40%;" alt="logo" src="{{ asset('img/logo/devmob.png') }}">
```